Bottom: c7f00b388188ec52687bac3e7318bab77d492528
Top:    ce6846556ec3739860b4debab7531350869b414d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-08 01:04:49 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2d60977..8f1f976 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -200,7 +200,6 @@ end_subcomposite (void)
   priv->subcomposite_stencil_bit = 0;
 }
 
-
 /* Compute group visibility based upon on copper layers only */
 static bool
 is_layer_group_visible (int group)
